HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -1012,206 +1014,206 @@ link::Guides/Tour-of-Special-Functions:: for plots showing ranges and asymptotes
 subsection:: Number Series
 Take a tour of link::Guides/Tour-of-Special-Functions#Number Series#Number Series::.
 
-method:: bernouliB2n (n)
+method:: bernouliB2n
```

Comment:
Do you disagree with the choice of functional notation? 

We discussed this before, and I responded with the changes you requested (110ca69), and functional notation is supported and documented in the `Writing Help` guide.

I can see how it gets around at least two of the problems I documented in #3694, but I also think that it makes the argument list less clear, in terms of how it corresponds to the boost function. For many of these, the arguments are conventions that will be recognizable to some people, and which are  obscured by receiver notation. I made mention of this in my original comment

For example `ibetaC` originally rendered as:
```
ibetaC(a,b, x)
   Returns the normalised complement of the incomplete beta function of a, b and x.
   ...
```
but now render as:
```
.ibetaC(b, x)
   Returns the normalised complement of the incomplete beta function of a, b and x.
   ...
```
so the description makes reference to an argument (receiver) that isn't documented.

Reply:
>Do you disagree with the choice of functional notation?

No, not at all, I would prefer that. But when rendering the help file, it fails the way you wrote it. I am sorry I had overlooked that it had been discussed already.

Reply:
To clarify, is the failure you're referring to the broken links in the table of contents? The failure in that case is in the help system renderer not the way this was written.